boards/arm/v2m_musca/v2m_musca-common.dtsi

```diff
@@ -0,0 +1,51 @@
+/*
```

Comment:
@galak @agross-linaro so, it seems to me that this file is included by _both_ the secure and the non-secure dts files.
Just to double-check; the reg properties (and corresponding Register Base Addresses) are the _same_ for Secure and Non-Secure access?? If so, this should work, otherwise, no.

And then, I have, again, the earlier comment; shouldn't this file be under ./dts/arm? Cause it is not really board specific.

Reply:
it may or may not be.  It depends on how the SOC vendors do their base board support and board specific add-ons.  In the case of this specific board, it's a one off board with a specific implementation.  So i suppose it could go in dts/arm, but it'll only be used for 1 board.  The v2m_musca-a.  If the musca-b is the same, then there is an argument for dts/arm.  But we haven't seen the memory map.

Reply:
this is required due to the ranges property being used to steer the addressing of the peripherals.